docs/cli/index.mdx

```diff
@@ -95,7 +95,6 @@ The following table lists and describes the environment variables you can set fo
 | `TEMPORAL_ADDRESS`                       | Host and port (formatted as host:port) for the Temporal Frontend Service. | --address                       |
 | `TEMPORAL_CODEC_AUTH`                    | Sets the authorization header for requests to Codec Server.               | --codec-auth                    |
```

Comment:
All the others start with the important noun, not a a description of use. To be consistent:
```suggestion
| `TEMPORAL_CODEC_AUTH`                    | Authorization header for requests to Codec Server.               | --codec-auth                    |
```